modules/cluster-logging-loki-deploy.adoc

```diff
@@ -92,10 +92,11 @@ spec:
     secret:
       name: logging-loki-s3
       type: s3
-  storageClassName: gp2
+  storageClassName: "<storage_class_name>" <1>
   tenants:
     mode: openshift-logging
 ----
+<1> Enter the name of an existing storage class for temporary storage. For best performance, specify a storage class that allocates block storage. Available storage classes for your cluster can be listed using `oc get storageclasses`.
```

Comment:
In the preview it looks like the following "Apply the configuration" block is indented too much, but I don't know what to add to the document to indent it less...